src/dynamics/ads_player/ads_player.js

```diff
@@ -437,7 +438,8 @@ Scoped.define("module:Ads.Dynamics.Player", [
                                 this._video.src = URL.createObjectURL(blob);
                                 this._video.crossOrigin = "anonymous";
                                 this._video.muted = true;
-                                return this._video.play()
+                                this._video.setAttribute("playsinline", true);
```

Comment:
Why would this attribute change to begin with?

Reply:
https://webkit.org/blog/6784/new-video-policies-for-ios/
https://stackoverflow.com/questions/75680106/how-to-play-a-video-in-the-background-on-ios-safari
safari ios has some security policy that causes the video to open in full screen

@jwellborn-kargo

Reply:
oh, i think i misunderstood the question, its because this._video is not the same variable as video